source/slang/hlsl.meta.slang

```diff
@@ -22416,13 +22416,290 @@ extension<T, L : IBufferDataLayout> RasterizerOrderedStructuredBuffer<T, L> : IR
     int getCount() { uint count; uint stride; this.GetDimensions(count, stride); return count; }
 }
 
+//
+// Cooperative Matrix enums
+//
+
+enum CoopMatMatrixUse
+{
+    MatrixA = 0,
+    MatrixB = 1,
+    MatrixAccumulator = 2,
+};
+
+enum CoopMatMatrixLayout
+{
+    RowMajor = 0,
+    ColumnMajor = 1,
+};
+
+enum CoopMatClampMode
+{
+    Undefined,
+    Constant,
+    ClampToEdge,
+    Repeat,
+    RepeatMirrored
+};
+
+namespace tensor_addressing
```

Comment:
assuming if the user is doing:
```
using namespace tensor_addressing;

TensorView view;
```

Lookup logic for `TensorView` will return two decls, one for `tensor_addressing::TensorView`, and one for `::TensorView`. And we need to make sure that when the `using namespace` declration exists, we disambiguate by preferring the decl in the deeper namespace over the global declaration.